CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6.3
+-fix visibility of data overlays on parent objects
```

Comment:
Please fix the formatting as per previous releases

Reply:
This is published here and looks like it is not working - https://grafana.com/grafana/plugins/grafana-iot-twinmaker-app/?tab=changelog

Reply:
Fixed

Reply:
it was missing the space after the Dashes